data/xbar_main.hjson

```diff
@@ -66,8 +66,8 @@
       reset: "rst_sys_ni",
       xbar:  false,
       addr_range: [{
-        base_addr: "0x81000000",
-        size_byte: "0x00001000",
+        base_addr: "0x80100000",
+        size_byte: "0x00000100",
```

Comment:
I would leave this as `0x1000`. If not, please make sure that UART and SPI can also fit in `0x100` and change all of this in the documentation as well.